src/status/status-module.ts

```diff
@@ -237,17 +240,19 @@ export default class StatusModule {
   }
 
   public async getStatus(withEcs = false) {
-    const [gitlab, runners, postgresql, screenshotter, systemHook] = await Promise.all([
+    const [gitlab, runners, postgresql, screenshotter, systemHook, charlesVersion] = await Promise.all([
       this.getGitlabStatus(),
       this.getRunnersStatus(),
       this.getPostgreSqlStatus(),
       this.getScreenshotterStatus(),
       this.getSystemHookStatus(),
+      getCharlesVersion(),
     ]);
 
     const response = {
       charles: {
         active: true,
+        version: charlesVersion ? charlesVersion : `Unknown (only available in production & staging)`,
```

Comment:
Could be: `charlesVersion || 'Unknown (only available in production & staging)'`